File: Bindings~/java/src/main/java/com/alttester/Commands/AltCommands/AltResetInput.java
/*
    Copyright(C) 2026 Altom Consulting
*/

package com.alttester.Commands.AltCommands;

import com.alttester.AltMessage;
import com.alttester.Commands.AltBaseCommand;
import com.alttester.IMessageHandler;

public class AltResetInput extends AltBaseCommand {
  public AltResetInput(IMessageHandler messageHandler) {
    super(messageHandler);
  }

  public String Execute() {
    AltMessage altMessage = new AltMessage();
    altMessage.setCommandName("resetInput");
    SendCommand(altMessage);
    return recvall(altMessage, String.class);
  }
}

File: Assets/AltTester/Runtime/AltDriver/Commands/AltCommands/AltResetInput.cs
/*
    Copyright(C) 2026 Altom Consulting
*/

using AltTester.AltTesterSDK.Driver.Commands;

namespace AltTester.AltTesterSDK.Driver.Commands
{
    public class AltResetInput : AltBaseCommand
    {

        public AltResetInput(IDriverCommunication communicationHandler) : base(communicationHandler) { }
        public void Execute()
        {
            var cmdParams = new AltResetInputParams();
            this.CommHandler.Send(cmdParams);
            var data = this.CommHandler.Recvall<string>(cmdParams);
            ValidateResponse("Ok", data);
        }
    }
}
